fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMutualTest.java

```diff
@@ -135,6 +135,48 @@ void testMutualIndexingNoBoundaries() {
         assertAllValidated(indexes);
     }
 
+    @Test
+    void testMutualIndexingNoBoundariesMultiIndexers() {
+        // Let multi threads build all the indexes - boundaries will be detected automatically - which means (null, null) because the data set will be too small to have multiple shards in fdb
+        // Note that some of the indexers should get conflicts, which should trigger the anyJumper mechanism. This test asserts that some indexers jump and abort at the third iteration.
+        final FDBStoreTimer timer = new FDBStoreTimer();
+
+        List<Index> indexes = new ArrayList<>();
+        indexes.add(new Index("indexA", field("num_value_2"), EmptyKeyExpression.EMPTY, IndexTypes.VALUE, IndexOptions.UNIQUE_OPTIONS));
+        indexes.add(new Index("indexB", field("num_value_3_indexed"), IndexTypes.VALUE));
+        indexes.add(new Index("indexC", field("num_value_unique"), EmptyKeyExpression.EMPTY, IndexTypes.VALUE, IndexOptions.UNIQUE_OPTIONS));
+
+        long numRecords = 180;
+        populateData(numRecords);
+
+        FDBRecordStoreTestBase.RecordMetaDataHook hook = allIndexesHook(indexes);
+        openSimpleMetaData(hook);
+        disableAll(indexes);
+
+        IntStream range = IntStream.rangeClosed(0, 7);
+        AtomicInteger count = new AtomicInteger(0);
+        range.parallel().forEach(ignore -> {
```

Comment:
Since there's only one fragment, this won't actually jump ever, right? Wouldn't we want a test that has like 5 ranges and 20 workers?

Reply:
Added. 
However, used 7 workers instead of 20. 
With 20 workers, some of them will not start indexing until the index becomes readable. You had mentioned the need for a framework that supports big db, this may be another use case.

Reply:
Update: 
I had to remove this test. Occasionally it fails (a matter of timing, that we can't control).